app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -53,14 +53,23 @@ def render_create_error_missing_email
     end
 
     def render_create_success
+      message = if Devise.paranoid
```

Comment:
I don't like this identation, we waste a lot of space, but if you prefer it this way, we can keep it

Reply:
@MaicolBen I'm fine with changing it!